packages/gdl-frontend/components/Reader/FontSizeContext.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2019 GDL
+ *
+ * See LICENSE
+ */
+
+import * as React from 'react';
+
+const FontSizeContext = React.createContext();
```

Comment:
To fix flow errors, you can type it like this:
```
type ContextState = {
  fontSize: number,
  changeFontSize: (newFontSize: number) => void
};

const FontSizeContext = React.createContext<ContextState>({
  fontSize: 14,
  changeFontSize: () => {}
});
```
Eventually
```
type ContextState = {
  fontSize: number,
  incrementFontSize: () => void,
  decrementFontSize: () => void,
};

const FontSizeContext = React.createContext<ContextState>({
  fontSize: 14,
  incrementFontSize: () => {},
  decrementFontSize: () => {},
});
```
`React.createContext(defaultValue)` require default value, which now works because what we missed yesterday was initializing the `Provider` on the top level.